gazebo_plugins/src/gazebo_ros_planar_move.cpp

```diff
@@ -111,6 +111,17 @@ namespace gazebo
     {
       odometry_rate_ = sdf->GetElement("odometryRate")->Get<double>();
     }
+    cmd_timeout_ = -1;
+    if (!sdf->HasElement("cmdTimeout"))
+    {
+      ROS_WARN_NAMED("planar_move", "PlanarMovePlugin (ns = %s) missing <cmdTimeout>, "
```

Comment:
It feels a bit weird to warn about not setting an optional parameter. All current users will start seeing this warning as soon as they update, even though there's nothing wrong with their setup. I think this should be and `INFO`... But I see it is following the pattern from the rest of the file, so I think it's ok to leave it like this too...

Reply:
As you prefer